docs/LED-INDICATION.md

```diff
@@ -22,3 +22,16 @@ that deserve user attention.
 | 12  | Controller replied without TLS connection state. |
 | 13  | Controller replied without OCSP response. |
 | 14  | Failed to fetch or verify Controller certificate. |
+
+# Appstatus display using LEDs
+
+Starting from commit e31ec53c0dbec104e4818c998223449576585d6b, application status is displayed using LEDs.
+That commit only supports model SIEMENS AG.SIMATIC IPC127E
+Uses LED 3 (the one labeled as L3 MAINT) for application state.
+
+1) If no application has even started booting that LED will be off.
```

Comment:
Avoid ")" with numbers in markdown, more info [here](https://www.markdownguide.org/basic-syntax/#ordered-list-best-practices)